data/core/units/elves/Fighter.cfg

```diff
@@ -15,7 +15,7 @@
     advances_to=Elvish Captain,Elvish Hero
     cost=14
     usage=fighter
-    description= _ "Elves are not warlike by nature, but in times of need, their natural grace and agility serve them well, as does their skillful craftsmanship. An elf can grasp the basics of swordsmanship and archery in a time uncannily short by the standards of other races, and put them to effective use on the battlefield."
+    description= _ "Among elves, swordplay is most commonly practiced by rangers and border patrols, who serve primarily as guardians of the forests and more occasionally as small squads for intrusions into other territories. Elvish swordsmanship is known for its emphasis on swift, agile strikes, generating less power than human or orcish warriors, but also leaving less time to react and counter. While difficult to master, most elves can learn the basics of swordplay relatively quickly and will readily pick up the blade when confronted with battle."
```

Comment:
more occasionally?

Perhaps rework the bit about difficult to master but most will pick up a sword? I think you're trying to say that while many elves will wield a sword, there are few who truly master it?